scimath/Mathematics/FitToHalfStatistics.tcc

```diff
@@ -128,6 +128,7 @@ void FitToHalfStatistics<CASA_STATP>::_getRealMinMax(
         CountedPtr<AccumType>& realMin, CountedPtr<AccumType>& realMax,
     CountedPtr<AccumType> knownMin, CountedPtr<AccumType> knownMax
 ) {
+    _setRange();
```

Comment:
is it ever wrong to call _setRange in these functions?
it only actually executes once as it has a fast path if it was already called
there is the chance that there is some other (maybe future) way of executing this stuff where it might again not have been called.
If the call is always correct as its either needed or a no-op, it might be safer to just call it everywhere its results are needed.